README.md

```diff
@@ -150,7 +150,7 @@ Remap PGs off of the given source OSD spec(s), up to the given maximum number of
 If a source OSD is included among target OSDs, it will be removed from the targets.
 
 ```
-$ ./pgremapper drain <osdspec>[,<osdspec>] --target-osds <osdspec>[,<osdspec>] [--allow-movement-across <bucket type>] [--max-backfill-reservations default_max[,osdspec:max]] [--max-source-backfills <n>]
+$ ./pgremapper drain <osdspec> [<osdspec> ...] --target-osds <osdspec>[,<osdspec>] [--allow-movement-across <bucket type>] [--max-backfill-reservations default_max[,osdspec:max]] [--max-source-backfills <n>]
```

Comment:
This is different from how we wrote this for `export-mappings`, at least, so a general harmonization would be appreciated.

Reply:
looks like a couple things slipped through